Weather/screens/HomeScreen.tsx

```diff
@@ -12,25 +14,49 @@ const HomeScreen = () => {
     
     const { username } = userContext;
 
-
     return(
         <View style={styles.container}>
             <Text>Welcome to the Home Screen!</Text>
             <Text style={styles.welcomeText}>Welcome {username}</Text>
```

Comment:
This code looks like it was left from a merge conflict, the return is already below this.